src/Command/SiteDbImportCommand.php

```diff
@@ -225,5 +225,32 @@ protected function execute(InputInterface $input, OutputInterface $output) {
     else {
       throw new SiteCommandException($shellProcess->getOutput());
     }
+
```

Comment:
We have to run at the end after the install command is executed - we can move into a method to clean it up a bit